diesel_cli/README.md

```diff
@@ -4,6 +4,34 @@ Diesel CLI
 Diesel CLI is a tool that aids in managing your database schema. Migrations are
 bi-directional changes to your database that get applied sequentially.
 
+Installation
+---------------
+
+The diesel cli, by default, requires [`libpq`][postgres], [`sqlite`][sqlite]
+and [`mysql`][mysql], some of which may already be included on your system. See
+which dependencies are already present with:
+
+```sh
+which psql
+which sqlite3
+which mysql
+```
+
+Once these dependencies are installed, you can run `cargo install diesel_cli`.
+
+To install the cli without these dependencies, omit the unneeded dependencies from
+the following command:
+
+```
+cargo install diesel_cli --no-default-features --features postgres,sqlite,mysql
+```
+
+The argument passed to `--features` must include no spaces, and be comma-separated.
```

Comment:
This works? Interesting! I've only ever seen the space-separated `"postgres sqlite mysql"` syntax, which is also what [cargo install --help](https://github.com/rust-lang/cargo/blob/305bc25d5e105e84ffe261655b46cf74570f6e5b/src/bin/install.rs#L55) mentions. Where did you see this?